nucypher/datastore/datastore.py

```diff
@@ -84,7 +84,7 @@ class Datastore:
 
     # LMDB has a `map_size` arg that caps the total size of the database.
     # We can set this arbitrarily high (1TB) to prevent any run-time crashes.
-    LMDB_MAP_SIZE = 1_000_000_000_000
+    LMDB_MAP_SIZE = 10_000_000
```

Comment:
We only wanted the lower map size for tests, no?

Reply:
Yeah, so what shall I do?

Reply:
I think @tuxxy had a `TestLMDBEnv` that could be used... @tuxxy any thoughts here?

Reply:
Use a session fixture

Reply:
Is there such a fixture as yet?

Reply:
I think it is already taken care of by @tuxxy - https://github.com/nucypher/nucypher/blob/main/tests/integration/conftest.py#L94